### PR TITLE
Update account

### PR DESCRIPTION
**Pull request by bigomise**
### 1. Objective

Add support of account API

### 2. Description of change

- added new resource responsible for account API call.
- updated index.d.ts
- added automatic tests

### 3. Quality assurance
update account using the following code example:
```
const omise = require('omise')({
    publicKey: 'pkey_test_XXXXXX',
    secretKey: 'skey_test_XXXXXX',
    omiseVersion: '2019-05-29'
  });
 omise.account.updateAccount({ webhook_uri: 'https://my/web/hooks' }, function(err, account) {
    console.log(account);
  });
```
check if account are updated.

or use auto-tests - run `npm test`

### 4. Impact of the change

N/A

### 5. Priority of change

Normal

### 6. Additional Notes

N/A

### CODE REVIEW

**Comment by rezigned** [comment]
@bigomise I think we should separate code formatting to another PR. It obscures the actual changes.

File: test/test_omise_account.js [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
       });
     });
+
+    it('should be able to update an account', function (done) {
+      testHelper.setupMock('account_update');
+      omise.account.updateAccount({
+        'webhook_uri': 'https://omise-flask-example.herokuapp.com/webhook',
+      }, function (err, resp) {
+
+        console.log(err, resp);
```
Comment (jacstn) [inline test/test_omise_account.js:25]: please consider to remove console.log, and space.

Comment (bigomise) [reply test/test_omise_account.js]: thanks.. removed.

**Comment by bigomise** [comment]
> @bigomise I think we should separate code formatting to another PR. It obscures the actual changes.

@rezigned i've tried to run `npm run jscs` that is suggested by our [document](https://github.com/omise/omise-node#contributions) but it seems it needs a configuration for `jscs`. 

 i would like to keep the original format but it was changed by my vscode 😂 .
Could you suggest me on this krub?

**Comment by rezigned** [comment]
@bigomise Ok krub 😆  . Let's merge :)

**Comment by bigomise** [comment]
thanks a lot krub @rezigned however, i don't have permission to merge it krub 😂. 
if you have a permission to do, please do for me na krub 🙇  thank you again krubbbbb.

**Pull request merged**